armi/physics/fuelCycle/fuelHandlers.py

```diff
@@ -90,6 +92,11 @@ def r(self):
         """Link to the Reactor object."""
         return self.o.r
 
+    def preoutage(self):
+        self.prepCore()
```

Comment:
Make `prepCore`, and `prepShuffleMap` private if you intend folks to just call `preoutage`. It is confusing if we have too many public methods. Additionally, please a detailed docstring on what this does exactly. What happens if I call `outage` before `preoutage`? Will this fail? If so, is there is a test for this?

Reply:
`prepCore` is intended to be an auxiliary function which can be defined by the user. Should it be set to private? 

`preoutage` is primarily used to store the locations of all assemblies before they are shuffled which is important for repeat shuffling, shuffling reports, and mcnp shuffling. I guess if this was called after `outage`, then it would impact all of those things. I don't think it would fail, but it would definitely screw with those things. Would it make sense to make these private as well? Both of those are called by `manageFuel`, and I don't see a need for anyone to call them individually.

Reply:
A good rule of thumb is if it CAN be private, it should be.

If the method/function isn't being called anywhere outside this file/class (ignoring tests), then it can be private.

Reply:
Set `prepCore` to private.